### DISCUSSION TITLE
Welcome to api-framework Discussions!

### DISCUSSION THREAD

**Discussion by jonnydgreen** [Eyrie-Tech/api-framework#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jonnydgreen** [comment]
Hey everyone, I'm Jonny - lead engineer, board game fanatic, and passionate about the climate!

**Comment by aleccool213** [comment]
Hey everyone, I am Alec :) Lead community facilitator (I just made that up!) and software developer. Excited about the future of Deno, JSR and API frameworks.
